micromasters_import/management/commands/queries/002_courserun_enrollment.sql

```diff
@@ -0,0 +1,65 @@
+--courserun migration from MicroMaster to MITxOnline
+INSERT INTO public.courses_courserun(
+    title,
+    courseware_id,
+    run_tag,
+    courseware_url_path,
+    start_date,
+    end_date,
+    enrollment_start,
+    enrollment_end,
+    expiration_date,
+    live,
+    created_on,
+    updated_on,
+    course_id)
+SELECT
+    mm_courserun.title,
+    mm_courserun.edx_course_key,
+    LEFT(CASE WHEN position('+' in mm_courserun.edx_course_key) > 0
+    THEN (regexp_match(mm_courserun.edx_course_key, '[^+]+$'))[1]
+    ELSE (regexp_match(mm_courserun.edx_course_key, '[^/]+$'))[1] END, 10) AS run_tag,
+    mm_courserun.enrollment_url,
+    mm_courserun.start_date,
+    mm_courserun.end_date,
+    mm_courserun.enrollment_start,
+    mm_courserun.enrollment_end,
+    mm_courserun.enrollment_end + INTERVAL '1 day',
+    NOT mm_courserun.is_discontinued,
+    coalesce(mm_courserun.enrollment_start, now()), --- need to confirm the default if enrollment_start is null
+    coalesce(mm_courserun.enrollment_end, now()),
```

Comment:
I don't understand the logic here. Why would we use `enrollment_start` for the record creation date?

Reply:
@pdpinch I don't recall specifically why I chose enrollment_start, but created is required field so we needed a similar field to populate it

Reply:
@pdpinch 
Here are available fields from MicroMaster courseruns
![image](https://user-images.githubusercontent.com/3138890/182205567-1a67299e-0b11-4674-836d-8f0a8591f143.png)
How should we populate created_on and updated_on? As you can see, there are blank for enrollment_start, start_date, enrollment_end and end_date in MicroMaster, so we need default value in case of blank for import

Reply:
I think this was a mapping we decided on in a call @rachellougee, and as you said, they're more or less arbitrary, I don't believe we use them in any user-facing logic or UI. The only place these values would show up is `django-admin`.

Reply:
Ok